src/PrestaShopBundle/Resources/config/services/adapter/admin.yml

```diff
@@ -1,13 +1,16 @@
 services:
   _defaults:
-    public: true
+    public: false
 
-  prestashop.adapter.admin.page_preference:
-    class: PrestaShop\PrestaShop\Adapter\Admin\PagePreference
+  PrestaShop\PrestaShop\Adapter\Admin\PagePreference:
     arguments:
       - "@session"
       - !php/const _PS_MODE_DEV_
     decorates: prestashop.core.admin.page_preference_interface
+
+  prestashop.adapter.admin.page_preference:
+    alias: PrestaShop\PrestaShop\Adapter\Admin\PagePreference
+    deprecated: ~
```

Comment:
No need for manual `public: false` on this service if you set it globally on the file